File: client-order-service/src/main/java/com/alekseytyan/client/dto/ClientLoadDTO.java
package com.alekseytyan.client.dto;

import com.alekseytyan.client.dto.enums.LoadStatus;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ClientLoadDTO implements Serializable {

    private String clientId;

    private String cityLoad;
    private String cityUnload;
    private String name;
    private int weight;

    private LoadStatus status;

    private String token;

}

File: telegram-bot-service/src/main/kotlin/com/alekseytyan/telegrambot/dto/ClientLoadDTO.kt
package com.alekseytyan.telegrambot.dto

import com.alekseytyan.telegrambot.dto.enums.LoadStatus
import lombok.AllArgsConstructor
import lombok.Data
import lombok.NoArgsConstructor
import java.io.Serializable

@Data
@AllArgsConstructor
@NoArgsConstructor
class ClientLoadDTO : Serializable {
    private val clientId: String? = null
    private val cityLoad: String? = null
    private val cityUnload: String? = null
    private val name: String? = null
    private val weight = 0
    private val status: LoadStatus? = null
    private val token: String? = null


    override fun toString(): String {
        return "Name: %s\n" +
                "City load: %s\n" +
                "City unload: %s\n" +
                "Weight: %d\n" +
                "Status: %s\n".format(name, cityLoad, cityUnload, weight, status)
    }
}